### ISSUE TITLE
Modify the "indicatif" help input information

### ISSUE THREAD

**Issue by zuluzaire** [litomba/www#212]
## Description

On the ad creation and modify profile and on board pages, for the telephone inputs modify the hint description message and the error message regarding the indicatif.
Add an example using brackets after the word "indicatif du pays".
"indicatif du pays (Ex:+237)"

## Steps to reproduce / User Story

1. Modify the hint of the ad creation page
2. Modify the hint of the on boarding page
3. Modify the hint of the telephone certification page
4. Modify the hint of the modify my profile page
5. Modify the error message of the ad creation page
6. Modify the error message of the on boarding page
7. Modify the error message of the telephone certification page
8. Modify the error message of the modify my profile page

### Operating System

- Android
- iOS
- Windows
- macOS

### Browser

- Chrome
- Firefox
- Safari
- Edge
- Opéra 
- Internet Explorer

**Comment by AntonioVdlC** [comment]
I think it's more user friendly to simply accept '00' for phone numbers ... right now I'm simply replacing it by a '+'.

**Comment by zuluzaire** [comment]
The replacement doesnt work correctly

I typed 2 zero fast, one has been replaced, the other one wasn't.
I felt like it was a bug on the site so I carried on adding 0 and that's what I get:
![image](https://user-images.githubusercontent.com/30872324/31142062-13bd9f46-a871-11e7-8937-930c4b31d2d1.png)


To me we should mention that the indicatif is + and not zero in the hints and in the error message, i think it's easier for us:
If you really want to switch the 0 to +, we still need to write it down in the description so the user gets not confused.
But as i said, i'm not fan of this in term of user experience.
